api/src/auth/drivers/ldap.ts

```diff
@@ -304,15 +301,18 @@ export class LDAPAuthDriver extends AuthDriver {
 				};
 			}
 
+			const schema = await this.getCurrentSchema();
```

Comment:
Same for the other usages but wont make a comment for each one 😄 
```suggestion
			const schema = await getSchema();
```